package.json

```diff
@@ -28,8 +28,12 @@
   },
   "homepage": "https://github.com/companieshouse/sic-code-web#readme",
   "dependencies": {
+    "@companieshouse/node-session-handler": "^5.0.1",
     "@companieshouse/structured-logging-node": "^1.0.4",
+    "@companieshouse/web-security-node": "^4.1.1",
     "axios": "^1.6.7",
+    "chai": "^5.1.1",
```

Comment:
chai is a testing library so should be in `devDependencies` - and appears to be there already so you can delete this line

Reply:
thanks @JoelClemence I have removed chai dependencies. Unit tests, lint and code are running fine.